integration_test/conf/openid4vci_frontend.yaml

```diff
@@ -75,6 +75,8 @@ config:
       session:
         timeout: 6
 
+  default_target_backend: "OpenID4VP"
```

Comment:
default target backed is not related to one specific frontend configuration but entire proxy configuration and must be set here:

https://github.com/italia/iam-proxy-italia/blob/refactor/directory/iam-proxy-italia-project/conf/microservices/target_based_routing.yaml#L4

please do not reimplement feature already implemented  in satosa, we just need to use them as they are